start.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 #
-# Start script for lfp-pay-web
+# Start script for pps-pay-web
 
 APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 if [[ -z "${MESOS_SLAVE_PID}" ]]; then
     source ~/.chs_env/private_env
     source ~/.chs_env/global_env
-    source ~/.chs_env/lfp-pay-web/env
+    source ~/.chs_env/pps-pay-web/env
```

Comment:
penalty-payment-web

Reply:
Changed from pps-pay-web to penalty-payment-web